src/main/java/org/opentripplanner/util/SemanticHash.java

```diff
@@ -0,0 +1,94 @@
+package org.opentripplanner.util;
```

Comment:
I'd rather move it under the Index API, since this is something we probably want to deprecate/move to the sandbox, and not include in the new API, would that be ok?

Reply:
done